### ISSUE TITLE
RRO 3.2.2 Windows: " in Rprofile.site files causes invalid multibyte character parser error

### ISSUE THREAD

**Issue by j-martens** [RevolutionAnalytics/RRO#239]
When starting RRO, console returns message: 
```
Error: invalid multibyte character in parser at line 15.
```
Source of error comes from 'Rprofile.site' in the {R_HOME}/etc.

It cannot print the character “.

For add'l info, user shared session information.
```
> sessionInfo()
R version 3.2.2 (2015-08-14)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 7 x64 (build 7601) Service Pack 1

locale:
[1] LC_COLLATE=Chinese (Traditional)_Taiwan.950 
[2] LC_CTYPE=Chinese (Traditional)_Taiwan.950   
[3] LC_MONETARY=Chinese (Traditional)_Taiwan.950
[4] LC_NUMERIC=C                                
[5] LC_TIME=Chinese (Traditional)_Taiwan.950    

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     
```

**Comment by j-martens** [comment]
Temporary Workaround: User substituted `` for the " in the `Rprofile.site` file and the error disappeared.

**Comment by nathansoz** [comment]
It looks like my copy/paste from outlook picked up a "smart-quote." I'll be fixing on the dev branch.

**Comment by nathansoz** [comment]
I'm surprised this didn't get picked up from our testing. Any other windows users see this ( @richcalaway or @sfweller ). I wonder if it is specific to some locales or versions of windows...

**Comment by ChingChuan-Chen** [comment]
I'm 陳慶全. This bug had come out in RRO-3.2.1, but I did not try to fix it and continue to use RRO-3.2.0. Although it does not cause any big trouble, I can't implement script with R function `source` or use windows command to run Rscript. Maybe the test does not go through this?

**Comment by nathansoz** [comment]
So I have the bad Rprofile.site on my system and I am able to do the following from the windows command line:

c:\Program Files\RRO\R-3.2.2\bin>Rscript -e "5+5"
[1] 10

I'm guessing this is an odd Locale/Unicode corner-case that didn't get picked up in our testing. I'll let @j-martens speak to what we want to do in the short-term, but it will be fixed for sure in the next release of RRO. I want to thank you for letting us know about this issue and apologize for any trouble it may have caused.
